mlos_bench/mlos_bench/tests/config/cli/test_load_cli_config_examples.py

```diff
@@ -54,6 +54,7 @@ def filter_configs(configs_to_filter: List[str]) -> List[str]:
 @pytest.mark.parametrize("config_path", configs)
 def test_load_cli_config_examples(config_loader_service: ConfigPersistenceService, config_path: str) -> None:
     """Tests loading a config example."""
+    # pylint: disable=too-complex
```

Comment:
Remove, unnecessary?

Reply:
Nope, it is.  Though I'm not sure why it's showing up here since I didn't change it ...